package.json

```diff
@@ -33,8 +33,10 @@
     "lint": "twolfson-style lint lib/ test/",
     "pretest": "npm run build",
     "test": "npm run verify-webdriver && mocha --timeout 10000 --reporter dot && npm run lint",
+    "test-windows": "mocha --timeout 10000 --reporter dot && npm run lint",
```

Comment:
Shouldn't this be
`npm run pretest && mocha --timeout 10000 --reporter dot && npm run lint`